scripts/find_versions_from_git_tags.py

```diff
@@ -3,10 +3,14 @@
 import re
 import os
 from datetime import datetime
+from dateutil.relativedelta import relativedelta
 from packaging import version as packaging_version
 
 versions = []
 
+LTS_VERSIONS = [
+    # "v3.4.1.0"
```

Comment:
@alexanderahn just FYI, any time you want to add a new EM release you'll have to add it to this list here (similar to the commented out line)